tools/clang/test/CMakeLists.txt

```diff
@@ -92,3 +97,13 @@ add_lit_target("check-clang-unit" "Running lit suite clang-unit"
           ARGS ${CLANG_TEST_EXTRA_ARGS}
         )
 # HLSL Change End - Add a separate target for clang unit tests
+
+# HLSL Change Begin - Add taef tests
+add_lit_target("check-clang-hlsl" "Running lit suite clang-hlsl"
```

Comment:
by convention the `check-clang-*` targets refer to the directory containing the lit suite. This is important because there is tooling to auto-generate build targets based on the subdirectory. This name will conflict with an existing subdirectory under tests, so it should probably be `taef` instead.